--- wkhtmltopdf-0.12.2.1/qt/src/3rdparty/webkit/Source/WebCore/html/HTMLImageElement.cpp	2015-01-18 09:09:30.000000000 +0000
+++ patches-0.12.2.1/HTMLImageElement.cpp	2024-10-15 20:01:46.067006011 +0000
@@ -74,7 +74,7 @@
     RefPtr<HTMLImageElement> image = adoptRef(new HTMLImageElement(imgTag, document));
     if (optionalWidth)
         image->setWidth(*optionalWidth);
-    if (optionalHeight > 0)
+    if (optionalHeight)
         image->setHeight(*optionalHeight);
     return image.release();
 }
